### PR TITLE
chore: remove kafka and druid implementation

### PR DESCRIPTION
**Pull request by cre8ivejp**
As part of migrating to BigQuery.
I will remove Kafka and Druid's helm manifests in another PR.

#### Things done

- Removed Druid implementation
- Removed Kafka implementation
- Removed Druid and Kafka settings from the helm manifest
- Removed Analysis dashboard

### CODE REVIEW

File: pkg/eventpersister/persister/persister.go [hunk 1]
```diff
@@ -317,19 +313,6 @@ func (p *Persister) send(messages map[string]*puller.Message) {
 			}
 			evs[id] = eventJSON
```
Comment (masaaania) [inline pkg/eventpersister/persister/persister.go:314]: We can delete `evs`, `eventJSON`, and the process of marshal event.

Comment (cre8ivejp) [reply pkg/eventpersister/persister/persister.go]: I have removed it.
https://github.com/bucketeer-io/bucketeer/pull/192/commits/73d315dca1127282c8a055b0c7a94532ce6d9dd0#diff-ad00cb580cd315d8619de51e511dc06052ace57d6146c30b60366e3a97b56d6cL291-L303

**Comment by cre8ivejp** [comment]
@masaaania @ono-max 
I have pushed the last changes. Please take a look when you have time.

**Review by masaaania — APPROVED** [review]
Great! 🎉

**Review by ono-max — APPROVED** [review]
👍

**Pull request merged**